### PR TITLE
Made the code Python 3 compatible

### PR DESCRIPTION
**Pull request by s-sanyal**

### CODE REVIEW

**Comment by cclauss** [comment]
Not quite right...

$ __python2 -c "for x in 'abc': print (x),"__
```
a b c
```
$ __python3 -c "for x in 'abc': print (x),"__
```
a
b
c
```

See #405

**Comment by cclauss** [comment]
Now it works great in Python 3 but it is now a syntax error in Python 2.  You need to add __from \_\_future\_\_ import print_function__ at the top of the file before any other imports.

**Comment by harshildarji** [comment]
@cclauss @s-sanyal It still gives syntax error while executing with Python 2!

File: data_structures/Stacks/Stock-Span-Problem.py [hunk 1]
```diff
@@ -37,7 +38,7 @@ def calculateSpan(price, S):
 # A utility function to print elements of array 
 def printArray(arr, n): 
     for i in range(0,n): 
-        print arr[i], 
+        print (arr[i],end =" ")
```
Comment (cclauss) [inline data_structures/Stacks/Stock-Span-Problem.py]: Please delete the space...  __print (arr[i],end =" ")__ --> __print(arr[i], end=" ")__

__print()__ is now a function like all other functions so it should be formatted in the same way.

**Comment by cclauss** [comment]
@ harshildarji Do you have `from __future__ import print_function` at the top of your file above all other imports?  What is the full text of the error that you are seeing?

**Comment by harshildarji** [comment]
@cclauss The problem was in the explanation part at the very beginning of the script. But now it's solved, so I'm merging this PR now.

**Pull request merged**